### DISCUSSION TITLE
Create images failed: we can't create your images right now

### DISCUSSION THREAD

**Discussion by AndreyTolstoy** [xtekky/gpt4free#1645] [Q&A]
I was generating images, but at some point they stopped being generated. I threw the error and got this:
An error occurred while generating the image: Create images failed: we can't create your images right now

What to do?

![clipboard_image_a8a91dc39929fcbc](https://github.com/xtekky/gpt4free/assets/161819069/a3b845ff-0244-4e5c-8498-ff9f05a4897d)

**Comment by hlohaus** [comment]
You need more tokens for Create Images with Bing, if this message appear. After some time, you become new tokens.
